bin/_static.js

```diff
@@ -4,13 +4,16 @@
 
 const config = require('../lib/config').getProperties();
 const logger = require('../lib/logging')('bin._static');
-const server = require('../lib/server/_static').create();
+
+async function create() {
+  const server = await require('../lib/server/_static').create();
+  server.start().then(() => {
```

Comment:
should use `await server.start();` here , move the log out of `then`. Just like below: https://github.com/mozilla/fxa-profile-server/pull/334/files#diff-8d44631bfd02d64ee75348ab92f466c0R27